pydra/engine/tests/test_satraexample.py

```diff
@@ -227,16 +360,38 @@ def test_8(plugin):
     assert results[0][1].output.out == 6
 
 
+# @pytest.mark.parametrize("plugin", Plugins)
+# def test_10a(plugin):
+#     """Test workflow with node level splitters and combiners"""
+#     wf = Workflow(name="test10", input_spec=["x"])
+#     wf.add(add2(name="add2", x=wf.lzin.x).split("x"))
```

Comment:
just a note that for this situation, the final state of the workflow itself would need to be updated, such that subsequent nodes can do the right thing.